packages/openshift/base_verbs.py

```diff
@@ -1422,9 +1424,12 @@ def node_ssh_client(apiobj_node_name_or_qname=None,
                 if not password:
                     password = cur_context().get_ssh_password()
 
+                if not key_filename:
+                    key_filename = None
+
```

Comment:
This check really isn't necessary.  The default value will always be **None**.  If something else is passed in, then an exception will be raised to the user.

Reply:
np, removed